{
  "cve": "CVE-2023-43782",
  "mitre": {
    "cpes": [],
    "created": "2023-09-22T00:00:00+00:00",
    "description": "Cadence through 0.9.2 2023-08-21 uses an Insecure /tmp/.cadence-aloop-daemon.x Temporary File. The file is used even if it has been created by a local adversary before Cadence started. The adversary can then delete the file, disrupting Cadence.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2023/43xxx/CVE-2023-43782.json",
    "references": [
      "http://www.openwall.com/lists/oss-security/2023/10/05/4",
      "https://bugzilla.suse.com/show_bug.cgi?id=1213983",
      "https://github.com/falkTX/Cadence"
    ],
    "title": null,
    "updated": "2024-09-24T18:19:55.663000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:falktx:cadence:*:*:*:*:*:*:*:*"
    ],
    "created": "2023-09-22T06:15:10.557000+00:00",
    "description": "Cadence through 0.9.2 2023-08-21 uses an Insecure /tmp/.cadence-aloop-daemon.x Temporary File. The file is used even if it has been created by a local adversary before Cadence started. The adversary can then delete the file, disrupting Cadence.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.5,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2023/CVE-2023-43782.json",
    "references": [
      "http://www.openwall.com/lists/oss-security/2023/10/05/4",
      "https://bugzilla.suse.com/show_bug.cgi?id=1213983",
      "https://github.com/falkTX/Cadence"
    ],
    "title": null,
    "updated": "2023-10-26T17:59:42.910000+00:00",
    "vendors": [
      "falktx",
      "falktx$PRODUCT$cadence"
    ],
    "weaknesses": [
      "CWE-668"
    ]
  },
  "opencve": {
    "changes": [
      {
        "created": "2024-09-24T19:30:00+00:00",
        "data": [
          {
            "details": {
              "added": {
                "ssvc": {
                  "options": {
                    "Automatable": "no",
                    "Exploitation": "poc",
                    "Technical Impact": "partial"
                  },
                  "version": "2.0.3"
                }
              },
              "removed": {},
              "updated": {}
            },
            "type": "metrics"
          }
        ],
        "id": "eb043b38-71a8-4ee5-8415-98419f7bcb14"
      }
    ],
    "cpes": {
      "data": [
        "cpe:2.3:a:falktx:cadence:*:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2023-09-22T00:00:00+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "Cadence through 0.9.2 2023-08-21 uses an Insecure /tmp/.cadence-aloop-daemon.x Temporary File. The file is used even if it has been created by a local adversary before Cadence started. The adversary can then delete the file, disrupting Cadence.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 5.5,
          "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
        },
        "provider": "nvd"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {
          "options": {
            "Automatable": "no",
            "Exploitation": "poc",
            "Technical Impact": "partial"
          },
          "version": "2.0.3"
        },
        "provider": "vulnrichment"
      },
      "threat_severity": {
        "data": null,
        "provider": null
      }
    },
    "references": {
      "data": [
        "http://www.openwall.com/lists/oss-security/2023/10/05/4",
        "https://bugzilla.suse.com/show_bug.cgi?id=1213983",
        "https://github.com/falkTX/Cadence"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    },
    "title": {
      "data": null,
      "provider": null
    },
    "updated": {
      "data": "2024-09-24T18:19:55.663000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "falktx",
        "falktx$PRODUCT$cadence"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-668"
      ],
      "providers": [
        "nvd"
      ]
    }
  },
  "vulnrichment": {
    "cpes": [],
    "created": "2023-09-22T00:00:00+00:00",
    "description": "Cadence through 0.9.2 2023-08-21 uses an Insecure /tmp/.cadence-aloop-daemon.x Temporary File. The file is used even if it has been created by a local adversary before Cadence started. The adversary can then delete the file, disrupting Cadence.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {},
      "kev": {},
      "ssvc": {
        "options": {
          "Automatable": "no",
          "Exploitation": "poc",
          "Technical Impact": "partial"
        },
        "version": "2.0.3"
      }
    },
    "references": [],
    "title": null,
    "updated": "2024-09-24T18:19:51.763000+00:00",
    "vendors": [],
    "vulnrichment_repo_path": "2023/43xxx/CVE-2023-43782.json",
    "weaknesses": []
  }
}